<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDBatchChangeVariableDialog</class>
 <widget class="QDialog" name="CMDBatchChangeVariableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="batchOriginalDataTypeBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Boolean</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Char</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Byte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Datetime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>原始数据类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="batchProjectDataTypeBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Boolean</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Char</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Byte</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UInt64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Datetime</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>String</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>工程数据类型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="batchDeviceListCombox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设备：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前变量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量组：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="curVarNameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前变量地址：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="curVarAddrEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量地址(*)：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="batchVarAddrEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="batchVarNameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量名(*)：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="batchVarCntEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量个数：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="batchVarNameStartEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="batchVarAddrStartEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="batchGroupListCombox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>431</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="nameFmt10Check">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10进制</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="nameFmt16Check">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>16进制</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="addrFmt10Check">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10进制</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="addrFmt16Check">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>16进制</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
